apps/monitoring/dynatrace/dynatrace-operator/dynatrace-operator-helmrepo.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: source.toolkit.fluxcd.io/v1beta1
+apiVersion: source.toolkit.fluxcd.io/v2beta1
```

Comment:
Please review, I dont think all need to be changed

Reply:
Updated and left the files with helm.toolkit.fluxcd.io in with Flux v2.